keps/sig-node/24-apparmor-ga/README.md

```diff
@@ -0,0 +1,604 @@
+  as node-level agents (e.g. specific version of CRI). Focus on external or
+  optional services that are needed. For example, if this feature depends on
+  a cloud provider API, or upon an external software-defined storage or network
+  control plane.
+
+  For each of the fill in the following, thinking both about running user
+  workloads and creating new ones, as well as about cluster-level services (e.g.
+  DNS):
+
+  This KEP adds no new dependencies.
+
+### Scalability
+
+_For alpha, this section is encouraged: reviewers should consider these questions
+and attempt to answer them._
+
+_For beta, this section is required: reviewers must answer these questions._
+
+_For GA, this section is required: approvers should be able to confirms the
+previous answers based on experience in the field._
+
+- **Will enabling / using this feature result in any new API calls?**
+
+  NO
+
+- **Will enabling / using this feature result in introducing new API types?**
+
+  NO
+
+- **Will enabling / using this feature result in any new calls to cloud
+  provider?**
+
+  NO
+
+- **Will enabling / using this feature result in increasing size or count
+  of the existing API objects?**
+
+  NO
+
+- **Will enabling / using this feature result in increasing time taken by any
+  operations covered by [existing SLIs/SLOs][]?**
+
+  NO
+
+- **Will enabling / using this feature result in non-negligible increase of
+  resource usage (CPU, RAM, disk, IO, ...) in any components?**
+  Things to keep in mind include: additional in-memory state, additional
+  non-trivial computations, excessive access to disks (including increased log
+  volume), significant amount of data send and/or received over network, etc.
+  This through this both in small and large cases, again with respect to the
+  [supported limits][].
+
+  NO
+
+### Troubleshooting
+
+Troubleshooting section serves the `Playbook` role as of now. We may consider
+splitting it into a dedicated `Playbook` document (potentially with some
+monitoring details). For now we leave it here though.
+
+_This section must be completed when targeting beta graduation to a release._
+
+- **How does this feature react if the API server and/or etcd is unavailable?**
+
+  This is integral part of both API server and the Kubelet. All their
+  dependencies will impact
+
+- **What are other known failure modes?**
+
+  No impact is being foreseen to running workloads based on the nature of
+  changes brought by this KEP.
+
+  Although some general errors and failures can be seen on [Failure and Fallback
+  Strategy](#failure-and-fallback-strategy).
+
+* **What steps should be taken if SLOs are not being met to determine the problem?**
+
+  N/A
+
+[supported limits]: https://git.k8s.io/community//sig-scalability/configs-and-limits/thresholds.md
+[existing slis/slos]: https://git.k8s.io/community/sig-scalability/slos/slos.md#kubernetes-slisslos
+
+## Implementation History
+
+- 2020-01-10: Initial KEP
```

Comment:
Can you update the implementation history here to mention the alpha and beta 1.4 releases (and the initial very old design proposal: https://github.com/kubernetes/kubernetes/pull/29168/files) following the timeline from https://github.com/kubernetes/enhancements/issues/24

Reply:
Added in [`24ca438`](https://github.com/kubernetes/enhancements/pull/3298/commits/24ca43823dcff7d7c9b7d5a8466c900ecad814c3)